### PR TITLE
 chore: Add New Resources

### LINKED ISSUE #940: [Add]  Add New Resources
### What would you like to share or ask?

I would like to include the following resources to the project.

* Images
1) Free Photos : https://freephotos.cc/en
2) Moose : https://icons8.com/photos
3) Burst : https://burst.shopify.com/
4) Barn Images : https://barnimages.com/
5) Vecteezy : https://www.vecteezy.com/
6) Pikwizard : https://pikwizard.com/

* Icons
1) CSS Icons : https://css.gg/
2) Feather Icons : https://feathericons.com/
3) IconHub : https://iconhub.io/

* Fonts
1) FontShare : https://www.fontshare.com/

* Animations
1) AnimXYZ : https://animxyz.com/
2) Animista : https://animista.net/

* Database
1) SupaBase : https://supabase.com/
2) SurrealDB : https://surrealdb.com/

* Youtube (Web Dev)
1) Codevolution : https://www.youtube.com/@Codevolution/videos
2) Easy Tutorials : https://www.youtube.com/@EasyTutorialsVideo/videos
3) Coding Nepal : https://www.youtube.com/@CodingNepal/videos
4) Mr.Web Designer : https://www.youtube.com/@MrWebDesignerAnas/videos
5) CodingLab : https://www.youtube.com/@CodingLabYT/videos
6) dotWebDesign : https://www.youtube.com/@dotWebdesigns/videos
7) Codewithsadee : https://www.youtube.com/@codewithsadee/videos

* Youtube (Web3)
1) Alchemy : https://www.youtube.com/@AlchemyPlatform/videos
2) Daulat Hussain : https://www.youtube.com/@daulathussain/videos

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by Shiva-Sai-ssb**
## Fixes Issue

Closes #940

## Changes proposed

I have included the following resources in their respective categories.

* Images
1) Free Photos : https://freephotos.cc/en
2) Moose : https://icons8.com/photos
3) Burst : https://burst.shopify.com/
4) Barn Images : https://barnimages.com/
5) Vecteezy : https://www.vecteezy.com/
6) Pikwizard : https://pikwizard.com/

* Icons
1) CSS Icons : https://css.gg/
2) Feather Icons : https://feathericons.com/
3) IconHub : https://iconhub.io/

* Fonts
1) FontShare : https://www.fontshare.com/

* Animations
1) AnimXYZ : https://animxyz.com/
2) Animista : https://animista.net/

* Database
1) SupaBase : https://supabase.com/
2) SurrealDB : https://surrealdb.com/

* Youtube (Web Dev)
1) Codevolution : https://www.youtube.com/@Codevolution/videos
2) Easy Tutorials : https://www.youtube.com/@EasyTutorialsVideo/videos
3) Coding Nepal : https://www.youtube.com/@CodingNepal/videos
4) Mr.Web Designer : https://www.youtube.com/@MrWebDesignerAnas/videos
5) CodingLab : https://www.youtube.com/@CodingLabYT/videos
6) dotWebDesign : https://www.youtube.com/@dotWebdesigns/videos
7) Codewithsadee : https://www.youtube.com/@codewithsadee/videos

* Youtube (Web3)
1) Alchemy : https://www.youtube.com/@AlchemyPlatform/videos
2) Daulat Hussain : https://www.youtube.com/@daulathussain/videos

### CODE REVIEW

File: database/youtube/web3-metaverse.json [hunk 1]
```diff
@@ -1,66 +1,82 @@
...
-        "language": "english"
-    }
-  ]
+  {
+    "name": "Code Eater",
+    "description": "This channel provides quality content for the most demanding topics like Web3, Blockchain & Metaverse in a very simple, easy to understand manner in Hindi language.",
+    "url": "https://www.youtube.com/@CodeEater21",
+    "category": "youtube",
+    "subcategory": "web3-metaverse",
+    "language": "english"
```
Comment (CBID2) [inline database/youtube/web3-metaverse.json:8]: [Proofreading]
```suggestion
    "language": "hindi"
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @Shiva-Sai-ssb! :)  I pointed out a proofreading error in my inline comment. In the future, consider checking if you’re describing the source accurately before making a pull request. Other than that, you're off to a great start! :)

**Comment by Shiva-Sai-ssb** [comment]
Hello @CBID2 , I believe I have successfully resolved the conflicts in the branch. Since this is my first time handling such a task, I'm unsure if I have done it correctly. Please inform me if the conflicts have been resolved. Additionally, I did not include the CodeEater channel, and the changes occurred automatically. However, I made the changes as per your request.

**Review by ujjawaltyagii — APPROVED** [review]
Lgtm! Reviewing this whole lot of resources was a task in itself o_o

**Review by CBID2 — APPROVED** [review]
LGTM!

**Review by rupali-codes — APPROVED** [review]
lgtm too, thanks a lot!

**Pull request merged**